unity/src/main/java/io/delta/unity/UCCatalogManagedCommitter.java

```diff
@@ -75,11 +76,33 @@ public CommitResponse commit(
 
     final CommitMetadata.CommitType commitType = commitMetadata.getCommitType();
 
-    if (commitType != CommitMetadata.CommitType.CATALOG_WRITE) {
-      // TODO: Support CATALOG_CREATE in the future, too
-      throw new UnsupportedOperationException("Unsupported commit type: " + commitType);
+    if (commitType == CommitMetadata.CommitType.CATALOG_CREATE) {
+      return createImpl(engine, finalizedActions, commitMetadata);
     }
+    if (commitType == CommitMetadata.CommitType.CATALOG_WRITE) {
+      return writeImpl(engine, finalizedActions, commitMetadata);
+    }
+
+    throw new UnsupportedOperationException("Unsupported commit type: " + commitType);
+  }
 
+  /** Handles CATALOG_CREATE by writing the published delta file for version 0. */
+  private CommitResponse createImpl(
+      Engine engine, CloseableIterator<Row> finalizedActions, CommitMetadata commitMetadata)
+      throws CommitFailedException {
+    final FileStatus kernelPublishedDeltaFileStatus =
+        writePublishedDeltaFileForVersion0(engine, finalizedActions, commitMetadata);
+
+    return new CommitResponse(ParsedLogData.forFileStatus(kernelPublishedDeltaFileStatus));
```

Comment:
> Note that UC still needs to be contacted about 00.json in a post-commit RPC hook. This is outside the purview of Kernel.

Where does this happen / who is responsible for doing this? This is not immediately clear to me